jenkins/agent-ctl.sh

```diff
@@ -54,15 +54,15 @@ usage() {
 set -o nounset
 set -o errexit
 
-DOCKER_VERSION='1.11.2-0~jessie'
+UNUSED_DOCKER_VERSION='1.11.2-0~jessie'
```

Comment:
yeah, if this is unused, no need to keep it here anymore. we can always find it in git history.